examples/kafka-to-pubsub/README.md

```diff
@@ -258,4 +155,4 @@ if (options.getOutputFormat() == FormatTransform.FORMAT.AVRO) {
     }
 ```
 
-_Note: The Kafka to Pub/Sub Dataflow Flex template supports SSL configuration with the certificate located only in GCS._
+_Note: The Kafka to Pub/Sub job supports SSL configuration with the certificate located only in GCS._
```

Comment:
I think it is better to highlight that when the job runs with a distributed runner it can take the certificate only from GCS:
_Note: The Kafka to Pub/Sub job executed with a distributed runner supports SSL configuration with the certificate located only in GCS._